exporter/azure_monitor/common/transport.go

```diff
@@ -35,12 +32,12 @@ func (e *Transporter) Transmit(o *Options, env *Envelope) {
 	defer reponse.Body.Close() // prevent possible resource leak
 
 	var result map[string]interface{}
-	
 	err = json.NewDecoder(reponse.Body).Decode(&result)
 	if err != nil {
 		fmt.Println("Error: check decoder\n")
 	}
+  
 	log.Println(result)
 	log.Println(result["data"])
-	fmt.Println("End Transmission")
-}
+	fmt.Println("End Transmission") // For debugging
+}
```

Comment:
empty line before EOF.

Reply:
I will add an empty line as the last line for all files.